### PR TITLE
Sync MAINTAINERS.md with reality on GitHub

### PR DESCRIPTION
**Pull request by liskin**

### CODE REVIEW

**Comment by liskin** [comment]
@psibi @peti @dmwit I linked to your Twitter but I'm not sure if that's a good idea. Let me know if I should delete it.

**Comment by psibi** [comment]
No objections from my side.

File: MAINTAINERS.md [hunk 1]
```diff
@@ -10,11 +10,24 @@
 
   * Daniel Wagner [GitHub][dmwit], IRC: `dmwit`
 
-  * David Lazar [GitHub][davidlazar]
+  * Peter Simons [GitHub][peti], [Twitter][twitter:peti]
 
-  * Devin Mullins [GitHub][twifkak]
+  * Sibi Prabakaran [GitHub][psibi], IRC: `psibi[m]`
```
Comment (psibi) [inline MAINTAINERS.md]: But I think you haven't linked to my twitter. :-)

**Comment by dmwit** [comment]
Sure, why not.

**Pull request merged**